crates/env/Cargo.toml

```diff
@@ -50,6 +50,27 @@ secp256k1 = { version = "0.24", features = ["recovery", "global-context"], optio
 rand = { version = "0.8", default-features = false, features = ["alloc"], optional = true }
 scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
 
+contract-metadata = "2.0.0-alpha.1"
+impl-serde = { version = "0.3.1", default-features = false }
+jsonrpsee = { version = "0.14.0", features = ["ws-client"] }
+pallet-contracts-primitives = "6.0.0"
+serde = { version = "1.0.137", default-features = false, features = ["derive"] }
+serde_json = "1.0.81"
+tokio = { version = "1.18.2", features = ["rt-multi-thread"] }
+log = "0.4"
+env_logger = "0.8"
+# TODO we need to use `subxt`'s `master` until the next release is published.
+subxt = { git = "https://github.com/paritytech/subxt" }
+
+# Substrate
+sp-rpc = "6.0.0"
+sp-core = "6.0.0"
+sp-keyring = "6.0.0"
+sp-runtime = "6.0.0"
+
+# TODO(#xxx) `smart-bench_macro` needs to be forked.
+smart-bench-macro = { git = "https://github.com/paritytech/smart-bench", branch = "cmichi-ink-e2e-test-mvp", package = "smart-bench-macro"}
+
```

Comment:
Do we really want to add all these dependencies to the `ink_env` crate? 

This might even merit a standalone so that if you don't want to use it then it won't bring in all these dependencies.

Reply:
A standalone crate will unfortunately have to depend on `ink_env`, due to the types (e.g. the `Environment` trait). This would introduce a cyclic dependency.